ixblue_c3_ins/include/ixblue_c3_ins/ixblue_c3_ins.h

```diff
@@ -0,0 +1,45 @@
+/*  QinetiQ North America (QNA)
+ *  350 Second Avenue
+ *  Waltham, MA 02451
+ *
+ *  Proprietary Licensed Information.
+ *  Not to be duplicated, used, or disclosed,
+ *  except under terms of the license.
+ *
+ *  Copyright © 2021 QinetiQ North America  All rights reserved.
+ */
+
+#ifndef IXBLUE_C3_INS_IXBLUE_C3_INS_H
+#define IXBLUE_C3_INS_IXBLUE_C3_INS_H
+
+#include <ros/ros.h>
+
+#include "ixblue_c3_ins/c3_protocol.h"
+
+
+namespace ixblue_c3_ins
+{
+
+class ixBlueC3InsDriver
+{
+public:
+  ixBlueC3InsDriver();
+  ~ixBlueC3InsDriver();
+
+  void spin();
+
+private:
+  void publish(const c3_protocol::nav_long::nav_long_data_t& data);
+
+  ros::NodeHandle nh_;
```

Comment:
Why do we have 2 nodeHandler?

Reply:
A public node handle and a private node handle. The public one is at the root namespace, and it is useful for topic interfaces ('/state'). The private one uses the fully qualified node name as namespace, and it is useful for parameters (`/ins_name/some_param`).

Reply:
Ok